server/views/components/new-site-notice/index.njk

```diff
@@ -3,14 +3,13 @@
     <div class="grid">
       <div class="grid__cell grid__cell--shift-l1">
         <div class="new-site-notice__inner">
-            <p class="new-site-notice__msg">
-                <span class="new-site-notice__device-ico">{% icon 'other/devices' %}</span>
-                <span class="new-site-notice__text">Welcome to the new reading experience for Wellcome Collection's narrative content.</span>
-            </p>
-            <div class="new-site-notice__ctas">
-                <a class="new-site-notice__cta" href="/articles/next-wellcome-website">Find out more</a>
-                <a class="new-site-notice__cta" href="?v=1">Take me back to the old site</a>
-            </div>
+          <p class="new-site-notice__msg">
+            <span class="new-site-notice__device-ico">{% icon 'other/devices' %}</span>
+            <span class="new-site-notice__text">Welcome to the new reading experience for Wellcome Collection's narrative content.</span>
+          </p>
+          <div class="new-site-notice__ctas">
+            <a class="new-site-notice__cta" href="/articles/next-wellcome-website">Find out more</a>
```

Comment:
I get a 404 error if I try to visit https://wellcomecollection.org/articles/next-wellcome-website.

Is that because none of this content is live yet, and the article will appear when this page is made public? Or is it just a broken link?